package.json

```diff
@@ -78,6 +81,7 @@
     "eslint-config-prettier": "^2.6.0",
     "eslint-plugin-node": "^5.2.0",
     "eslint-plugin-prettier": "^2.3.1",
+    "grpc": "^1.7.2",
```

Comment:
You can actually get `grpc` through the common-grpc module:

```js
var grpc = require('@google-cloud/common-grpc').grpc;
```

Reply:
Nice, thanks!